prusti-viper/src/encoder/backtranslation.rs

```diff
@@ -0,0 +1,457 @@
+use std::collections::HashMap;
```

Comment:
In the context of this PR it is clear what this is, but for posteriority let's make it `counterexample_translation.rs` and likewise `CounterexampleTranslator`.